taskun-scheduler/src/main/java/com/github/seratch/taskun/scheduler/impl/TaskunImpl.java

```diff
@@ -117,6 +117,11 @@ public boolean isRunning() {
         return cronInvocation.getCurrentRawCrontabLines();
     }
 
+    @Override
+    public long getLastInvocationTime() {
```

Comment:
Sorry to say this in despite of that this naming was my suggestion, getLastCronInvocationTimeMillis would be better here.